numpy/f2py/setup.py

```diff
@@ -26,6 +26,9 @@
 def configuration(parent_package='', top_path=None):
     config = Configuration('f2py', parent_package, top_path)
     config.add_subpackage('tests')
+    config.add_subpackage('backends')
```

Comment:
This is only needed if there's a `backends/setup.py`.

Reply:
I am not much familiar with `distutils` configuration files. Would it be more apt to use the `add_data_dir` function instead? With the current configuration, the `backend` folder will not be included in the installed `numpy/f2py` folder.

Reply:
`add_data_dir` is more appropriate. However, after looking at how it's done elsewhere, `add_subpackage` is used more often (even if it gives a warning). So you can leave it as is.